pkg/systemd/services/service.jsx

```diff
@@ -79,7 +79,7 @@ export class Service extends React.Component {
                       id="service-details"
                       breadcrumb={
                           <Breadcrumb>
-                              <BreadcrumbItem to={"#" + cockpit.location.href.replace(cockpit.location.path[0], '')}>{_("Services")}</BreadcrumbItem>
+                              <BreadcrumbItem to={"#/" + cockpit.location.href.replace(cockpit.location.encode(cockpit.location.path[0]), '')}>{_("Services")}</BreadcrumbItem>
```

Comment:
I feel like this is making an unnecessarily strong assumption about how the page encodes the URL. IMHO it would be simpler and more robust to just replace the entire `#.*(?|$)` with `#\1`, i. e. dropping the entire fragment for system services (no `?` in the URL), or dropping the unit name (keeping the `?`).